web-ts/jest.config.ts

```diff
@@ -24,5 +27,10 @@ module.exports = {
       }
     ]
   ],
-  snapshotSerializers: ['enzyme-to-json/serializer']
+  snapshotSerializers: ['enzyme-to-json/serializer'],
+  globals: {
+    __BUILD_CONFIG__: webIntegreatTestCmsBuildConfig
+  }
```

Comment:
as `__BUILD_CONFIG__` is a const we cannot (at least not without typescript error)  override it in the `jest.setup.ts` another solution is to override it right here. This will not throw any error (as jest probably does the parsing) and works perfectly fine (which means I tested it 👍🏼 )